lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -218,8 +231,13 @@ function dosomething_reportback_form($form, &$form_state, $entity = NULL) {
       )
     ),
     '#default_value' => $entity->quantity,
+    '#description' => t('Only type in the number -- like "300" or "5".'),
   );
 
+  if (!user_access('edit any reportback')) {
+    unset($form['reportback_inputs']['quantity']['#description']);
+  }
+
```

Comment:
instead of doing these if checks for each thing
can you have one if and unset it all at once?